packages/playwright-test/src/common/compilationCache.ts

```diff
@@ -27,7 +27,11 @@ export type MemoryCache = {
 };
 
 const version = 13;
-const cacheDir = process.env.PWTEST_CACHE_DIR || path.join(os.tmpdir(), 'playwright-transform-cache');
+
+const DEFAULT_CACHE_DIR_WIN32 = path.join(os.tmpdir(), `playwright-transform-cache`);
+const DEFAULT_CACHE_DIR_POSIX = path.join(os.tmpdir(), `playwright-transform-cache-` + os.userInfo().username);
```

Comment:
Are we sure `username` is safe to use in file paths?

Reply:
According to https://unix.stackexchange.com/a/157431/323684 - seems to be safe on Debian. However, I added sanitization - just in case.